redash/handlers/__init__.py

```diff
@@ -22,6 +23,14 @@ def status_api():
     return jsonify(status)
 
 
+@routes.route("/resource-proxy", methods=["GET"])
```

Comment:
Let's put this in the `api` namespace (i.e. `/api/resource-proxy`) to avoid issues in environments like our deployment preview.

Reply:
Done https://github.com/getredash/redash/pull/4599/commits/c7b13459e8f4da6251d801acaa6012a7d1f7fc78